src/components/shared/VirtualList.js

```diff
@@ -172,8 +172,14 @@ type VirtualListProps = {|
   +disableOverscan: boolean,
   +columnCount: number,
   +containerWidth: CssPixels,
-  +role?: string,
+  // by using these props we are effectively telling screen readers that
+  // the browser (and our code) will take care of all accessibilty issues (like focus, navigation)
```

Comment:
What do you think of: `The next 3 props will be applied to the underlying DOM element. They're important for accessibility (especially focus and navigation).`

(Also note: a sentence starts with a capital letter and ends with a dot :) )